src/main/java/com/puppycrawl/tools/checkstyle/checks/DescendantTokenCheck.java

```diff
@@ -186,6 +331,24 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public class DescendantToken {
+ *   public void exampleMethod() {
+ *       try {
+ *           // Some code
+ *       } catch (Exception e) {
+ *           System.out.println("Caught exception");
+ *           return; // Violation
+ *       } finally {
+ *           System.out.println("Finally block");
+ *           // OK
```

Comment:
In some examples we also put violation on multiple problems. So it is ok to put ```//OK```.please correct me if am wrong.

Reply:
`Violation ` just means on this there is some problem, it can be many.

Ok comment is usually a noise, but sometimes it is good to put it if there is bunch of code around and you need user attention only to single line.

----

I think I misread example, please just remove `//ok` comment line. No reason to point attention to it.

Reply:
from every example right ?

Reply:
At least from here. 
Others. Did not confuse me.